a/aopalliance-repackaged/2.4.0-b32_ubuntu_18.04.sh

```diff
@@ -0,0 +1,84 @@
+# ----------------------------------------------------------------------------
+#
+# Package          : aopalliance-repackaged_2.4.0-b32
+# Version          : 1.0.0
+# Source repo      : https://github.com/Appdynamics/OSS
+# Tested on        : ubuntu_18.04
+# Passing Arguments: 1.Version of package, 2.JDK version (openjdk-8-jdk or openjdk-11-jdk)
+# Script License   : Apache License, Version 2 or later
+# Maintainer       : Arumugam N S <asellappen@yahoo.com> / Priya Seth<sethp@us.ibm.com>
+#
+# Disclaimer       : This script has been tested in non-root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+
+
+export REPO=https://github.com/Appdynamics/OSS
+
+#no tag currently and using master by default
+if [ -z "$1" ]; then
+  export VERSION="origin/master"
+else
+  export VERSION="$1"
+fi
+
+#Default testing on jdk8
+if [ -z "$2" ]; then
+  export JDK="openjdk-8-jdk"
+else
+  export JDK="$2"
+fi
+
+
+sudo apt-get update
+sudo apt-get install wget git -y
+
+if [ -d "OSS" ] ; then
+  rm -rf OSS
+fi
+
+
+git clone ${REPO}
+
+
+## Build and test   aopalliance-repackaged/2.4.0-b32/
+cd  OSS/aopalliance-repackaged/2.4.0-b32/
+git checkout ${VERSION}
+ret=$?
+
+if [ $ret -eq 0 ] ; then
+  echo "$Version found to checkout "
+else
+  echo "$Version not found "
+  exit
+fi
+
+sudo apt install  -y maven
+
+
+# run tests with java 11 or jdk 8
+sudo apt-get install -y ${JDK}
+jret=$?
+if [ $jret -eq 0 ] ; then
+  echo "Sucessfully installed $JDK "
+else
+  echo "Failed to install $JDK "
+  exit
+fi
+
+
+#Setting JAVA_HOME
+export folder=`echo ${JDK}  | grep -oP '(?<=openjdk-).*(?=-jdk)'`
+export JAVA_HOME=/usr/lib/jvm/java-${folder}-openjdk-ppc64el/
+
+mvn -v
+
+## Build and test 2.4.0-b32
+cd aopalliance-repackaged/2.4.0-b32
```

Comment:
Isn't "cd" to this directory already done ? Also pl. update the logs of the build (preferably to a non-default version of the package)

Reply:
cd is duplicate and removed now.
captured log already for def and non def ,but missed to add here .

added now as below

sh 2.4.0-b32_ubuntu_18.04.sh
=====================


[INFO] --- maven-jar-plugin:2.4:jar (default-jar) @ aopalliance-repackaged ---
[INFO] Downloading from jvnet-nexus-releases: https://maven.java.net/content/repositories/releases/classworlds/classworlds/1.1-alpha-2/classworlds-1.1-alpha-2.jar
[INFO] Downloading from central: https://repo.maven.apache.org/maven2/classworlds/classworlds/1.1-alpha-2/classworlds-1.1-alpha-2.jar
[INFO] Downloaded from central: https://repo.maven.apache.org/maven2/classworlds/classworlds/1.1-alpha-2/classworlds-1.1-alpha-2.jar (38 kB at 2.2 MB/s)
[INFO] Adding existing MANIFEST to archive. Found under: /OSS/aopalliance-repackaged/2.4.0-b32/target/classes/META-INF/MANIFEST.MF
[INFO] Building jar: /OSS/aopalliance-repackaged/2.4.0-b32/target/aopalliance-repackaged.jar
[INFO]
[INFO] --- maven-install-plugin:2.4:install (default-install) @ aopalliance-repackaged ---
[INFO] Downloading from jvnet-nexus-releases: https://maven.java.net/content/repositories/releases/org/codehaus/plexus/plexus-utils/3.0.5/plexus-utils-3.0.5.pom
[INFO] Downloading from central: https://repo.maven.apache.org/maven2/org/codehaus/plexus/plexus-utils/3.0.5/plexus-utils-3.0.5.pom
[INFO] Downloaded from central: https://repo.maven.apache.org/maven2/org/codehaus/plexus/plexus-utils/3.0.5/plexus-utils-3.0.5.pom (2.5 kB at 167 kB/s)
[INFO] Downloading from jvnet-nexus-releases: https://maven.java.net/content/repositories/releases/org/codehaus/plexus/plexus/3.1/plexus-3.1.pom
[INFO] Downloading from central: https://repo.maven.apache.org/maven2/org/codehaus/plexus/plexus/3.1/plexus-3.1.pom
[INFO] Downloaded from central: https://repo.maven.apache.org/maven2/org/codehaus/plexus/plexus/3.1/plexus-3.1.pom (19 kB at 1.2 MB/s)
[INFO] Downloading from jvnet-nexus-releases: https://maven.java.net/content/repositories/releases/org/codehaus/plexus/plexus-digest/1.0/plexus-digest-1.0.pom
[INFO] Downloading from central: https://repo.maven.apache.org/maven2/org/codehaus/plexus/plexus-digest/1.0/plexus-digest-1.0.pom
[INFO] Downloaded from central: https://repo.maven.apache.org/maven2/org/codehaus/plexus/plexus-digest/1.0/plexus-digest-1.0.pom (1.1 kB at 59 kB/s)
[INFO] Downloading from jvnet-nexus-releases: https://maven.java.net/content/repositories/releases/org/codehaus/plexus/plexus-components/1.1.7/plexus-components-1.1.7.pom
[INFO] Downloading from central: https://repo.maven.apache.org/maven2/org/codehaus/plexus/plexus-components/1.1.7/plexus-components-1.1.7.pom
[INFO] Downloaded from central: https://repo.maven.apache.org/maven2/org/codehaus/plexus/plexus-components/1.1.7/plexus-components-1.1.7.pom (5.0 kB at 311 kB/s)
[INFO] Downloading from jvnet-nexus-releases: https://maven.java.net/content/repositories/releases/org/codehaus/plexus/plexus-utils/3.0.5/plexus-utils-3.0.5.jar
[INFO] Downloading from jvnet-nexus-releases: https://maven.java.net/content/repositories/releases/org/codehaus/plexus/plexus-digest/1.0/plexus-digest-1.0.jar
[INFO] Downloading from central: https://repo.maven.apache.org/maven2/org/codehaus/plexus/plexus-utils/3.0.5/plexus-utils-3.0.5.jar
[INFO] Downloading from central: https://repo.maven.apache.org/maven2/org/codehaus/plexus/plexus-digest/1.0/plexus-digest-1.0.jar
[INFO] Downloaded from central: https://repo.maven.apache.org/maven2/org/codehaus/plexus/plexus-digest/1.0/plexus-digest-1.0.jar (12 kB at 539 kB/s)
[INFO] Downloaded from central: https://repo.maven.apache.org/maven2/org/codehaus/plexus/plexus-utils/3.0.5/plexus-utils-3.0.5.jar (230 kB at 7.2 MB/s)
[INFO] Installing /OSS/aopalliance-repackaged/2.4.0-b32/target/aopalliance-repackaged.jar to /root/.m2/repository/org/glassfish/hk2/external/aopalliance-repackaged/2.4.0-b32/aopalliance-repackaged-2.4.0-b32.jar
[INFO] Installing /OSS/aopalliance-repackaged/2.4.0-b32/pom.xml to /root/.m2/repository/org/glassfish/hk2/external/aopalliance-repackaged/2.4.0-b32/aopalliance-repackaged-2.4.0-b32.pom
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  02:06 min
[INFO] Finished at: 2021-04-20T05:33:32Z
[INFO] ------------------------------------------------------------------------

oot@975ca5676aed:~/.m2/repository/org/glassfish/hk2/external/aopalliance-repackaged/2.4.0-b32# ls -ltr
total 28
-rw-r--r-- 1 root root  6770 Apr 20 05:09 aopalliance-repackaged-2.4.0-b32.pom
-rw-r--r-- 1 root root 14756 Apr 20 05:33 aopalliance-repackaged-2.4.0-b32.jar
-rw-r--r-- 1 root root   218 Apr 20 05:33 _remote.repositories


 sh 2.4.0-b32_ubuntu_18.04.sh  "" openjdk-11-jdk
================================


[INFO]
[INFO] --- maven-jar-plugin:2.4:jar (default-jar) @ aopalliance-repackaged ---
[INFO] Adding existing MANIFEST to archive. Found under: /OSS/aopalliance-repackaged/2.4.0-b32/target/classes/META-INF/MANIFEST.MF
[INFO] Building jar: /OSS/aopalliance-repackaged/2.4.0-b32/target/aopalliance-repackaged.jar
[INFO]
[INFO] --- maven-install-plugin:2.4:install (default-install) @ aopalliance-repackaged ---
[INFO] Installing /OSS/aopalliance-repackaged/2.4.0-b32/target/aopalliance-repackaged.jar to /root/.m2/repository/org/glassfish/hk2/external/aopalliance-repackaged/2.4.0-b32/aopalliance-repackaged-2.4.0-b32.jar
[INFO] Installing /OSS/aopalliance-repackaged/2.4.0-b32/pom.xml to /root/.m2/repository/org/glassfish/hk2/external/aopalliance-repackaged/2.4.0-b32/aopalliance-repackaged-2.4.0-b32.pom
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  18.307 s
[INFO] Finished at: 2021-04-20T05:51:14Z
[INFO] ------------------------------------------------------------------------
root@975ca5676aed:/#



created /OSS/aopalliance-repackaged/2.4.0-b32/target/aopalliance-repackaged.jar and unstalled at 
/root/.m2/repository/org/glassfish/hk2/external/aopalliance-repackaged/2.4.0-b32/aopalliance-repackaged-2.4.0-b32.jar


root@975ca5676aed:/# ls -ltr ./root/.m2/repository/org/glassfish/hk2/external/aopalliance-repackaged/2.4.0-b32/aopalliance-repackaged-2.4.0-b32.jar
-rw-r--r-- 1 root root 14758 Apr 20 05:51 ./root/.m2/repository/org/glassfish/hk2/external/aopalliance-repackaged/2.4.0-b32/aopalliance-repackaged-2.4.0-b32.jar
root@975ca5676aed:/# date
Tue Apr 20 05:52:58 UTC 2021
root@975ca5676aed:/# ls -ltr /OSS/aopalliance-repackaged/2.4.0-b32/target/aopalliance-repackaged.jar
-rw-r--r-- 1 root root 14758 Apr 20 05:51 /OSS/aopalliance-repackaged/2.4.0-b32/target/aopalliance-repackaged.jar
root@975ca5676aed:/#